using_images/s2i_images/php.adoc

```diff
@@ -124,6 +124,13 @@ The following environment variable sets its equivalent property value in the opc
 |16M
 |===
 
+ifdef::openshift-origin[]
+|`*OPCACHE_REVALIDATE_FREQ*`
+|How often to check script timestamps for updates, in seconds. 0 will result in OPcache checking for updates on every request.
```

Comment:
Could you maybe link somewhere upstream about this setting, so the users can better understand what it does?